spec/ddtrace/contrib/sinatra/tracer_spec.rb

```diff
@@ -646,6 +667,19 @@
       expect(span.get_tag(Datadog::Contrib::Sinatra::Ext::TAG_ROUTE_PATH)).to eq(url) if app_name == self.app_name
       expect(span.get_tag(Datadog::Contrib::Sinatra::Ext::TAG_SCRIPT_NAME)).to be_nil
       expect(span.span_type).to eq(Datadog::Ext::HTTP::TYPE_INBOUND)
+
+      expect(span.get_tag(Datadog::Ext::Metadata::TAG_COMPONENT))
+        .to eq(Datadog::Contrib::Sinatra::Ext::TAG_COMPONENT)
+
+      case span.name
+      when Datadog::Contrib::Sinatra::Ext::SPAN_REQUEST
+        expect(span.get_tag(Datadog::Ext::Metadata::TAG_OPERATION))
+          .to eq(Datadog::Contrib::Sinatra::Ext::TAG_OPERATION_REQUEST)
+      when Datadog::Contrib::Sinatra::Ext::SPAN_ROUTE
+        expect(span.get_tag(Datadog::Ext::Metadata::TAG_OPERATION))
+          .to eq(Datadog::Contrib::Sinatra::Ext::TAG_OPERATION_ROUTE)
+      end
```

Comment:
Consider adding here a `raise` for the else case, just in case in the future we extend the integration and forget to update this.